scripts/main.js

```diff
@@ -52,20 +52,32 @@ $appList.on('click', 'li', function (event) {
 $goBackButton.on('click', function () {
   setRoute('')
 })
-
-
 $editButton.on('click', function(event){
-$body.attr('data-state', 'update-app')
 var id = $('#name-app').data('id')
 setRoute(id + '/edit')
-handleEditRout()
```

Comment:
when we click the "edit" button, it’s just like clicking a link: all we want is to change the route. Everything else must happen in the edit rout handler